quedex_api/user_stream.py

```diff
@@ -104,6 +104,22 @@ def on_order_cancel_failed(self, order_cancel_failed):
     """
     pass
 
+  def on_all_orders_cancelled(self, all_orders_cancelled):
+    """
+    :param all_orders_cancelled: dummy parameter (possibly empty dictionary)
```

Comment:
Misleading comment, I wouldn't know what to expect here. Make it an empty dict always and add explanation that this is for future extensibility?

Reply:
Expect the unexpected! Personally I don't want this parameter here, but it is required by generic listener invoking mechanism. We just pass whole event dictionary here, which in this case contains is only {"type": "all_orders_cancelled"}. But there is nothing said about "type" key in other comments. So maybe i would just say "dummy parameter, reserved for future extension" ?